modules/03 Front End Development/Assignments/54/index.html

```diff
@@ -0,0 +1,260 @@
+                    init();
+                    startGame();
+                    this.parentNode.removeChild(this);
+                }
+            }
+            else {
+                var pName = document.getElementById("player" + player).value
+                document.getElementById("turn").innerText = pName + "'s turn";
+            }
+        }
+        function isEmpty(row, col) {
+            return positions[row][col].isEmpty;
+        }
+        function startGame() {
+            var p1 = document.getElementById("player1").value;
+            var p2 = document.getElementById("player2").value;
+            var round = Math.floor(Math.random() * 2) + 1;
+            var pName = round === 1 ? p1 : p2
+            document.getElementById("turn").innerText = pName + "'s turn";
+            turn = round;
+            initializeArray(3);
+        }
+        function check_board() {
+            for (var r = 0; r < 3; r++) {
+                for (var c = 0; c < 3; c++) {
+                    if (positions[r][c].isEmpty) {
+                        return false;
+                    }
+                }
+            }
+            return true;
+        }
+        function check_winning(row, col, player) {
+            var row_winning = winning_by_row(row, player);
+            var col_winning = winning_by_col(col, player);
+            var diagonal_winning = winning_by_diagonal(row, col, player);
+            return row_winning || col_winning || diagonal_winning;
+        }
+        function winning_by_row(row, player) {
+            var count = 0;
+            for (var i = 0; i < 3; i++) {
+                if (positions[row][i].player === player) {
+                    count++;
+                }
+            }
+            return count === 3;
+        }
+        function winning_by_col(col, player) {
+            var count = 0;
+            for (var i = 0; i < 3; i++) {
+                if (positions[i][col].player === player) {
+                    count++;
+                }
+            }
+            return count === 3;
+        }
+        function winning_by_diagonal(row, col, player) {
+            var count_diagonal_one = 0;
+            var count_diagonal_two = 0;
+            for (var i = 0; i < 3; i++) {
+                if (positions[i][i].player === player) {
+                    count_diagonal_one++;
+                }
+            }
+            for (var r = 0, c = 2; r < 3; r++ , c--) {
+                if (positions[r][c].player === player) {
+                    count_diagonal_two++;
+                }
+            }
+            if (count_diagonal_one === 3 || count_diagonal_two === 3) {
+                return true;
+            }
+            else return false;
+        }
+
+        
+        function changeColors(btn){
+            if(btn === "lightBtn"){
+                document.getElementsByTagName("body")[0].style.backgroundColor = "#FFD0CC"
```

Comment:
I'd prefer setting a single class instead of iterating through elements and modify their styles